Main.java

```diff
@@ -9,7 +10,20 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // create a scanner for user input
+    Scanner input = new Scanner(System.in);
+    
+    // ask user for the degrees celcius
+    System.out.println("Please enter the degrees celcius:");
+
+    // declare and initialize a variable for the degrees celcius
+    int degreesCelcius = input.nextInt();
+
+    int degreesFarenheit = (degreesCelcius * 9) / 5 + 32;
+
+    // tell the user the degrees farenheit
+    System.out.println("It is " + degreesFarenheit + " degrees farenheit outside.");
+
```

Comment:
Great work. The variable names and comments are fantastic. The reason you are failing the test cases is because your output does not match EXACTLY like the sample cases. Try and match these up so that it does pass. Otherwise, good work!